ontology/yaml/resources/HVAC/entity_types/CH.yaml

```diff
@@ -504,3 +504,34 @@ CH_US_SBO_901_1:
   - INCOMPLETE
   uses:
   - cooling_percentage_sensor
+
+CH_SS_CHWDT:
+  description: "Non-standard chiller."
+  implements:
+  - CH
+  - INCOMPLETE
+  - SS
+  - CHWDT
+  opt_uses:
+  - lost_power_alarm
+  - run_mode
+
+CH_SS_CHWSWISOVM_CWSISOVM_CWRISOVM_CHWRWISOVM_CHWDT:
+  description: "Non standard chiller with chilled/condensing supply/return water isolation valve control. Chilled water flowrate status "
+  implements:
+  - CH
+  - SS
+  - CHWSWISOVM
+  - CWSISOVM
+  - CWRISOVM
+  - CHWRWISOVM
+  - CHWDT
+  uses:
+  - outside_air_temperature_sensor
+  - chilled_water_flowrate_status
+  - condensing_water_flowrate_status
+  - condensing_supply_water_temperature_sensor
```

Comment:
belongs on an abstract. swtc or something.

Reply:
As discussed, since this is non-standard chiller, this was correct.